src/main/resources/com/nirima/jenkins/plugins/docker/DockerTemplate/help-credentialsId.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    The SSH credentials to connect to the instance with
```

Comment:
I'm not a native speaker, but I think it is not needed. Instance is also not correct because it's called container.